demos/clarad.js

```diff
@@ -0,0 +1,30 @@
+const axios = require('axios');
+
+// Print out a list of names in Gwen's repo. Can use other endpoints too, such as, id.
+// Make a call to the Github API chaining .then() and .catch() onto the Promises and print out your results.
+axios.get('https://api.github.com/users/gwenf/repos')
+    .then(response => {
+        response.data.forEach(element => console.log(element.name));
+    })
+    .catch(error => {
+        console.log(error);
+    });
+
+// Now create an async function and make the call using await syntax
+async function axiosAsync () {
+    // await response of fetch call
+    let response = await axios.get('https://api.github.com/users/gwenf/repos');
+    // only proceed once promise is resolved
+    let data = await response;
+    // only proceed once second promise is resolved
+    return data;
```

Comment:
If you remove line 18, you can just return `response.data` directly.

Reply:
Yes, I see that. I will remove line 18 and then do a new pull request. Thank you.

Reply:
Line 18 (actually lines 17 and 18) were removed (see below).